packages/@aws-cdk/aws-events-targets/lib/ecs-task-properties.ts

```diff
@@ -2,7 +2,7 @@ export interface ContainerOverride {
   /**
    * Name of the container inside the task definition
    */
-  readonly containerName: string;
+  readonly name: string;
```

Comment:
I would prefer for the parameter to be called `containerName`. My apologies if it renders incorrectly, but that could and should be a different concern.

Reply:
Ok, will update